test.js

```diff
@@ -204,3 +221,83 @@ test('`onEviction` option method is called after `maxSize` is exceeded', t => {
 	t.is(actualValue, expectValue);
 	t.true(isCalled);
 });
+
+test('max age should remove the item on get it again', async t => {
```

Comment:
Prefix all the `maxAge` tests.
```suggestion
test('maxAge - max age should remove the item on get it again', async t => {
```